content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -178,36 +231,36 @@ The requirements to run TrueCommand in Docker Desktop for Windows are:
 * CPU support for VM Monitor Mode Extension (VT-c on Intel CPUs).
 * Hyper-V is enabled in Windows 10.
 * 4 GB memory at minimum.
-* [Docker Desktop](https://www.docker.com/products/docker-desktop) needs to be installed in Windows.
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) must be installed in Windows.
 
 ### Enable Hyper-V
 
-To enable Hyper-V, click on Windows Start button and select or search for *Apps & Features*.
-Select *Programs and Features* under **Related settings** and *Turn Windows Features on or off* (Administrator action).
-Set the Hyper-V option and click *Ok*.
- You will need to restart your restart your computer for the change to take effect.
+Open the Start menu and search for Apps & Features.
+Select Programs and Features under Related settings and Turn Windows Features on or off (Administrator action).
```

Comment:
I can't tell what the UI elements are in this sentence. Even if it is another UI or software program, if it is on the screen it goes in bold. If Related settings and Turn Windows Features is the UI or on-screen element use bold, otherwise this instruction is a bit confusing.
```suggestion
Select **Programs and Features** under Related settings and Turn Windows Features on or off (Administrator action).
```